react/src/components/card/SprkCard.test.js

```diff
@@ -8,109 +7,39 @@ import SprkCard from './SprkCard';
 Enzyme.configure({ adapter: new Adapter() });
 
 describe('SprkCard:', () => {
```

Comment:
I love how much cleaner everything looks without the config objects in the code!

Do we want to add tests for `additionalClasses`?

Reply:
Thanks @bobwise ! Great idea, I will add the test and update HTML to use secondary.